File: onebot/src/main/java/cn/evole/onebot/sdk/response/guild/GetGuildMsgResp.java
package cn.evole.onebot.sdk.response.guild;

import cn.evole.onebot.sdk.event.message.GuildMessageEvent;
import com.google.gson.annotations.SerializedName;
import lombok.Data;


@Data
public class GetGuildMsgResp {

    @SerializedName("guild_id")
    public String guildId;

    @SerializedName("channel_id")
    public String channelId;

    @SerializedName("message")
    public String message;

    @SerializedName("message_id")
    public String messageId;

    @SerializedName("message_seq")
    public int messageSeq;

    @SerializedName("message_source")
    public String messageSource;

    @SerializedName("sender")
    private GuildMessageEvent.GuildSender sender;

    @SerializedName("time")
    public long time;

}

File: onebot/src/main/kotlin/sdk/response/guild/GetGuildMsgResp.kt
package cn.evolvefield.onebot.sdk.response.guild

import cn.evolvefield.onebot.sdk.entity.GuildSender
import com.google.gson.annotations.SerializedName
import lombok.Data

@Data
class GetGuildMsgResp {
    @SerializedName("guild_id")
    var guildId = ""

    @SerializedName("channel_id")
    var channelId = ""

    @SerializedName("message")
    var message = ""

    @SerializedName("message_id")
    var messageId = ""

    @SerializedName("message_seq")
    var messageSeq = 0

    @SerializedName("message_source")
    var messageSource = ""

    @SerializedName("sender")
    private val sender: GuildSender? = null

    @SerializedName("time")
    var time = 0L
}
